tool/net/redbean.c

```diff
@@ -3130,6 +3135,9 @@ static char *ServeStatusz(void) {
   AppendLong1("lastmeltdown", shared->lastmeltdown);
   AppendLong1("workers", shared->workers);
   AppendLong1("assets.n", assets.n);
+#ifndef STATIC
+  AppendLong1("lua.memory", lua_gc(L, LUA_GCCOUNT)*1024 + lua_gc(L, LUA_GCCOUNTB));
```

Comment:
This is really cool but please note it only reports Lua memory usage for the main process.

Reply:
Yes, that was the intent, to see the usage in the main process.